api/src/main/java/org/openmrs/module/fhir/api/impl/base/BaseConditionHelperImpl.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.fhir.api.impl.base;
+
+import ca.uhn.fhir.rest.server.exceptions.UnprocessableEntityException;
+import org.apache.commons.lang.ArrayUtils;
+import org.hl7.fhir.dstu3.model.Condition;
+import org.openmrs.Obs;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.fhir.api.util.FHIRConditionUtil;
+import org.openmrs.module.fhir.api.util.FHIRUtils;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+/**
+ * @see org.openmrs.module.fhir.api.helper.ConditionHelper
+ * @since 1.20.0
+ */
+
+public class BaseConditionHelperImpl {
+
+	/**
+	 * @see org.openmrs.module.fhir.api.helper.ConditionHelper#getCondition(String)
```

Comment:
Can you include the package of the `String` in the above `see` annotation parameter?